### ISSUE TITLE
Preview broken with latest version of Craft

### ISSUE THREAD

**Issue by nathansnelgrove** [spicywebau/craft-neo#451]
#### Description
I am not sure if this is a problem specific to my site somehow or not, but it looks like all the Neo fields on a client site of mine no longer preview properly. They will preview fine until a change is made, and then both the full-screen Draft Preview feature and the live AJAX entry preview fail to render anything in a Neo field.

#### Steps to reproduce
1. Create a Neo field and a Matrix field, and add them to an entry type.
2. Fill in all the fields in the entry and publish the entry.
3. Edit the entry in Live Preview mode. Make any change to any field. The Matrix field will load properly. The Neo field will not.

#### Other information
- Neo version: latest
- Craft version: latest

**Comment by nathansnelgrove** [comment]
Some additional notes:

Here is a simplified version of where I see this failure happening.

```
{% for block in entry.pageContent.level(1).all() %}
     {% switch block.type.handle %}
          {% case "panel" %}
           <section class="panel">
               {# Load up the section container #}
               <div class="panel-content">
                       {% for row in block.children.all() %}
                             {# Any code here does not load #}
                       {% else %}
                             <p>This is a failure mode, which always loads in Preview after making a page edit. Once the page is published, the rows above are visible (as they should be).</p>
                       {% endif %}
                </div><!--.panel-content-->
         </section><!--.panel-->

    {% endswitch %}
{% endfor %}

```

**Comment by nathansnelgrove** [comment]
I'll add this was also mentioned [in March](https://github.com/spicywebau/craft-neo/issues/305#issuecomment-598554832) with another user.

**Comment by ttempleton** [comment]
Looks like this started happening with the latest release and you should be safe to upgrade as far as 2.9.6 for now.  The comment you linked to described an unrelated issue #367 which was fixed in 2.7.17.  Will work on a fix for this issue which will be released in the next couple of days.
